app/code/Magento/Sitemap/Api/Data/SitemapInterface.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
```

Comment:
Please add return types to all getters

Reply:
Added nullable return types to all getters